.go-version

```diff
@@ -0,0 +1 @@
+1.21.9
```

Comment:
Please rollback this change, and let's update & discuss it under https://github.com/etcd-io/bbolt/pull/713

Reply:
Just to clarify, does the rollback of this change mean 
- to include the code changes made here with PR #713? 
Or 
- just to change the go version here?

Reply:
The latter. One PR is supposed to only do one thing, so let's do not update the go version in this PR.

Reply:
Fixed according to feedback! Thank you for clarifying! :)